packages/plugin-core/src/survey.ts

```diff
@@ -607,19 +608,13 @@ export class SurveyUtils {
             "https://airtable.com/shry4eLgvVE6WR0Or?prefill_SurveyName=InactiveFeedback";
           VSCodeUtils.openLink(AIRTABLE_URL);
 
-          await StateService.instance().updateGlobalState(
-            GLOBAL_STATE.INACTIVE_USER_SURVEY_SUBMITTED,
-            "submitted"
-          );
+          MetadataService.instance().setInactiveUserMsgStatus("submitted");
           vscode.window.showInformationMessage(
             "Thanks for helping us make Dendron better 🌱"
           );
           AnalyticsUtils.track(SurveyEvents.InactiveUserSurveyAccepted);
         } else {
-          await StateService.instance().updateGlobalState(
-            GLOBAL_STATE.INACTIVE_USER_SURVEY_SUBMITTED,
-            "cancelled"
-          );
+          MetadataService.instance().setInactiveUserMsgStatus("cancelled");
```

Comment:
nitpick: should "submitted" and "cancelled" here be an enum?

Reply:
`Metadata.inactiveUserMsgStatus` has a union type of `"submitted" | "cancelled"`. I can't think of any immediate benefits of making this an enum, but I don't have any strong opinions on this. Do we have a convention around this? I did add some conventions around using types and enums in the context of things we expose with config, but I don't think we can apply those generally.

Open to suggestions.

Reply:
so the *right* thing to do for all strings is to have them be localized instead of using raw strings. since its a user facing prompt, would not use an enum. for now, would just keep this as is

Reply:
This is not a user facing prompt. it is the metadata values that we set to store the state of the past prompts.

Reply:
ah, i see. in that case, enum makes sense

Reply:
switched them to an enum